srcpkgs/sslh/template

```diff
@@ -10,6 +10,7 @@ license="GPL-2.0-or-later"
 homepage="https://www.rutschle.net/tech/sslh/README.html"
 distfiles="https://github.com/yrutschle/sslh/archive/v${version}.tar.gz"
 checksum=2e457e59592f8e523cade8d9302b0fdc87f8ea0322beb674dd7f067547a93ea9
+patch_args=-Np1
```

Comment:
This is not necessary anymore.

Reply:
(we default to Np1)

Reply:
This patch is a merge from the last release until the patch from upstream, but only the files handle the config.